Dockerfile

```diff
@@ -89,6 +82,15 @@ RUN apk add --no-cache \
 
 FROM index.docker.io/library/alpine:latest
 
+LABEL maintainer="ClamAV bugs <clamav-bugs@external.cisco.com>"
+
+EXPOSE 3310
+EXPOSE 7357
+
+HEALTHCHECK CMD "clamdcheck.sh"
+
+ENTRYPOINT [ "/init" ]
```

Comment:
Not sure why you also moved the entry point; as the file does not exist at this place; so logically this is a bit murkey. Yes the statement is optimized in docker cache layers; but it potentially actually breaks things, as its undefined here. Moving the COPY up to here is also troublesome, because we don't have a container yet, and you don't want to risk anything getting over-written etc.

Reply:
Btw, @micahsnyder do you also want a 'clean' commit history, with just a single commit? I guess this change is trivial enough that a 'squash commits' merge would fix any issue, otherwise, I'd recommend @sammyhk to cleanup the commit history as well. But that's up to @micahsnyder :)

Reply:
I agree moving up the ENTRYPOINT statement is a bit murky as it is not yet exists at that point. Actually I'm thinking the same thing as the HEALTHCHECK statement as clamdcheck.sh is not yet exists. So I wondered if just move EXPOSE statement is enough or not

Reply:
You are correct, the healthcheck should probably stay there too :) well caught; I'll update that in my own Dockerfiles where applicable too :) I think I made that mistake elsewhere :)

Reply:
Clean them up, only move LABEL and EXPOSE statement after 2nd FROM statement. Moved HEALTHCHECK statement after the COPY statements. Cleaned the history as single commit.